Fw/Python/src/fprime/fbuild/cmake.py

```diff
@@ -536,7 +538,7 @@ def _communicate(proc, stdout, stderr, print_output=True):
                     line = key.fileobj.readline().decode().replace("\r\n", "\n")
                 # Some systems (like running inside Docker) raise an io error instead of returning "" when the device
                 # is ended. Not sure why this is, but the effect is the same, on IOError assume end-of-input
-                except IOError as ioe:
+                except IOError as dummy_ioe:
```

Comment:
Just remove the variable.  You can also use "_" to discard unused variable.

Reply:
Will do